src/components/Navbar/UserData.json

```diff
@@ -1,19 +1,19 @@
 [
```

Comment:
Let's move all static, constants to `/constants`

Reply:
Done